internal/app/filestorage/access.go

```diff
@@ -0,0 +1,64 @@
+package filestorage
+
+import (
+	"context"
+	"encoding/json"
+	"log/slog"
+)
+
+func (fs *Storage) Get(ctx context.Context, shortURL string) (string, error) {
+
+	return fs.cache.Get(ctx, shortURL)
+}
+
+func (fs *Storage) Push(ctx context.Context, short string, original string) error {
+	err := fs.cache.Push(ctx, short, original)
+	if err != nil {
+		slog.Error("Push to cache storage error", "err", err)
+		return err
+	}
+
+	item := makeStorageItem(short, original)
+
+	data, err := json.Marshal(item)
+	if err != nil {
+		slog.Error("When convert to json error", "err", err)
+		return err
+	}
+
+	data = append(data, '\n')
+
+	_, err = fs.file.Write(data)
+	if err != nil {
+		slog.Error("When write to file error", "err", err)
+		return err
+	}
+
+	return nil
+}
+
+func (fs *Storage) PushBatch(ctx context.Context, list *map[string]string) error {
```

Comment:
Так как такие типы как слайс и мапа и так передаются в методы по ссылке нет особого смысла использовать указатель.
```suggestion
func (fs *Storage) PushBatch(ctx context.Context, list map[string]string) error {
```